### PR TITLE
EVALSYS-1499 - Extend administrate-search to enable keyword searching…

### PR DESCRIPTION
**Pull request by jonespm**
… by Assign Group Name

### CODE REVIEW

File: sakai-evaluation-impl/src/java/org/sakaiproject/evaluation/logic/EvalDeliveryServiceImpl.java [hunk 1]
```diff
@@ -572,6 +572,15 @@ protected boolean checkAnswersValidForEval(EvalResponse response, boolean checkR
                 throw new IllegalArgumentException("NULL templateItem or templateItem.item for answer: " 
                         + "Answers must have the templateItem set, and that must have the item set");
             }
+
```
Comment (smarquard) [inline sakai-evaluation-impl/src/java/org/sakaiproject/evaluation/logic/EvalDeliveryServiceImpl.java]: This code change (in EvalDeliveryServiceImpl) doesn't appear to belong to this issue so shouldn't be part of this PR.

**Comment by bjones86** [comment]
Besides @smarquard's comment, there's a couple opportunities for diamond operators, but otherwise LGTM

File: sakai-evaluation-impl/src/java/org/sakaiproject/evaluation/logic/EvalDeliveryServiceImpl.java (hunk 1 above)
Comment (jonespm) [inline sakai-evaluation-impl/src/java/org/sakaiproject/evaluation/logic/EvalDeliveryServiceImpl.java]: Thanks must have been some weird rebase conflict that left it like this.
